scanner/src/main/java/ca/on/oicr/gsi/runscanner/scanner/processor/BaseOxfordNanoporeProcessor.java

```diff
@@ -63,27 +67,64 @@ public PathType getPathType() {
       Files.walkFileTree(
           root.toPath(),
           new FileVisitor<Path>() {
+
+            /**
+             * Actions to take before visiting each individual file within the directory, mostly
+             * dealing with directory path.
+             */
             @Override
             public FileVisitResult preVisitDirectory(
                 Path path, BasicFileAttributes basicFileAttributes) throws IOException {
-              // Immediately check if we're dealing with the old format, move on if so.
-              if (POSITION.matcher(path.toString()).find()) {
-                log.debug("Skipping " + path + " because we found a Sequencer Position in it.");
+              log.debug("Pre-visit: " + path);
+
+              /**
+               * If directory matches criteria we know exclude the directory from consideration,
+               * don't go into the directory (SKIP_SUBTREE)
+               */
+              if (excludedDirectoryFormat(path) || olderThanCutoff(path)) {
+                log.debug("Skipping " + path + " because we found an excluded directory in it.");
                 return FileVisitResult.SKIP_SUBTREE;
               }
+
+              /**
+               * If directory matches criteria we know include the directory, add without looking
+               * inside directory (add, SKIP_SUBTREE). Checking if result of readsDirectoryForRun
+               * 'isDirectory' is an existence check.
```

Comment:
I'm a bit lost reading this paragraph, specifically on `criteria we know include the directory` and  `Checking if result...is an existence check.` Can you clarify?

Reply:
Rewrote this comment

Reply:
Did the rewrite get pushed? It looks the same to me right now...

Reply:
Got eaten in a rebase, sorry! Should be good now